canonical-websites/jaas.ai
# Remove redundant scripts

## Done

Remove redundant script file and duplication

## QA

- Pull branch
- Run `./run`
- Check the site has no terminal errors on build
- View site at http://0.0.0.0:8029/
- Check all main sections and ensure no console errors
